docs/ref/settings.txt

```diff
@@ -809,11 +810,11 @@ DATAFILE_MAXSIZE
 
 .. versionadded:: 1.8
 
-Default: ``'500M'``
+Default: ``'500MB'``
```

Comment:
The existing documentation is correct: https://github.com/django/django/blob/3baebf52aa0850de3ebadcbe1eda7e6e9a2c6c44/django/db/backends/oracle/creation.py#L284